readthedocs/settings/base.py

```diff
@@ -270,13 +296,19 @@ def USE_PROMOS(self):  # noqa
             'options': {'queue': 'web'},
         },
     }
+    MULTIPLE_APP_SERVERS = [CELERY_DEFAULT_QUEUE]
+    MULTIPLE_BUILD_SERVERS = [CELERY_DEFAULT_QUEUE]
 
     # Sentry
     SENTRY_CELERY_IGNORE_EXPECTED = True
 
     # Docker
     DOCKER_ENABLE = False
+    DOCKER_SOCKET = 'unix:///var/run/docker.sock'
+    DOCKER_BUILD_IMAGES = None
```

Comment:
Maybe we can put a comment that this is deprecated in favor or `DOCKER_IMAGE_SETTINGS`

Reply:
Okay :100: